.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,119 @@
+            core.setOutput('contributor-username', pr.user.login);
+            
+      - name: Send bounty data to BLT
+        if: steps.check-issues.outputs.has-bounty == 'true'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const issueNumber = steps.check-issues.outputs['issue-number'];
+            const bountyAmount = steps.check-issues.outputs['bounty-amount'];
+            const contributorUsername = steps.check-issues.outputs['contributor-username'];
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)